k256/src/lib.rs

```diff
@@ -28,6 +28,8 @@
 
 #[cfg(feature = "arithmetic")]
 mod arithmetic;
+#[cfg(feature = "arithmetic")]
+pub use arithmetic::lincomb;
```

Comment:
I'd prefer to keep the re-export in just `arithmetic`:

```suggestion
```

Reply:
But `arithmetic` is a private module, it will be inaccessible.
I did move it to be with the other re-exports from this module

Reply:
Aah, my bad